src/app/public/src/modules/clipboard/clipboard.service.ts

```diff
@@ -43,18 +51,25 @@ export class StacheClipboardService {
     this.windowRef.nativeWindow.getSelection().removeAllRanges();
   }
 
-  private createCopyTextArea() {
-    const copyTextAreaStyles = `
-      font-size: 12px;
-      border: 0;
-      padding: 0;
-      margin: 0;
-      position: absolute;
-      left: -9999px;
-      top: ${this.windowRef.nativeWindow.pageYOffset || this.document.documentElement.scrollTop}px;`;
+  private createTextAreaFromElement(element: HTMLElement): HTMLTextAreaElement {
+    if (!this.copyTextArea) {
+        const copyTextAreaStyles = `
+        font-size: 12px;
+        border: 0;
+        padding: 0;
+        margin: 0;
+        position: absolute;
+        left: -9999px;
+        top: ${this.windowRef.nativeWindow.pageYOffset || this.document.documentElement.scrollTop}px;`;
```

Comment:
What's the reasoning behind calculating the `top` value, and not just setting it to something negative? `-100000px` or something?